playground/backend/cmd/server/controller.go

```diff
@@ -121,6 +121,15 @@ func (controller *playgroundController) GetCompileOutput(ctx context.Context, in
 	return &pipelineResult, nil
 }
 
+// Cancel is setting cancel flag to stop code processing
+func (controller *playgroundController) Cancel(ctx context.Context, info *pb.CancelRequest) (*pb.CancelResponse, error) {
+	pipelineId := info.PipelineUuid
+	if err := setToCache(ctx, controller.cacheService, uuid.MustParse(pipelineId), cache.Canceled, true); err != nil {
```

Comment:
Doesn't MustParse return an error?

Reply:
it throws panic if couldn't parse the `pipelineId` to uuid.
```
func MustParse(s string) UUID {
	uuid, err := Parse(s)
	if err != nil {
		panic(`uuid: Parse(` + s + `): ` + err.Error())
	}
	return uuid
}
```